lua/entities/magicbutton.lua

```diff
@@ -29,6 +29,12 @@ if (SERVER) then
             if (IsValid(button)) then
                 button:Spawn()
                 button:Activate()
+
+                if (BotSayGlobal) then
```

Comment:
just remove this kind of junk, if you need BotSayGlobal to work then you can define it yourself or ask me to move it to contrib